<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBoxModel1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>561</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Brownian Diffusion</string>
   </property>
   <widget class="QPushButton" name="pushButtonFormula_1">
    <property name="geometry">
     <rect>
      <x>464</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Formula</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam1_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam2_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelParam1_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelParam2_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>delta</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRelLikelihood_1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBIC_1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelRelLikelihood_1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rel. Likelihood</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBIC_1">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>BIC</string>
    </property>
   </widget>
  </widget>
  <widget class="ModelFitWidget" name="plot">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>561</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxModel2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>561</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Confined Diffusion</string>
   </property>
   <widget class="QPushButton" name="pushButtonFormula_2">
    <property name="geometry">
     <rect>
      <x>464</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Formula</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam1_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam2_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelParam1_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelParam2_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>delta</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRelLikelihood_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBIC_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelRelLikelihood_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rel. Likelihood</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBIC_2">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>BIC</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam3_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelParam3_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>tau</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBoxLogPlot">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>860</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Log Scale</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClipboard">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>460</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Results to Clipboard</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAnalyze">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>460</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Analyze</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxModel3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>561</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Hopping Diffusion</string>
   </property>
   <widget class="QPushButton" name="pushButtonFormula_3">
    <property name="geometry">
     <rect>
      <x>464</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Formula</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam1_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam2_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelParam1_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>D_macro</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelParam2_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>D_micro</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRelLikelihood_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBIC_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelRelLikelihood_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rel. Likelihood</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBIC_3">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>BIC</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam3_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelParam3_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>delta</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam4_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelParam3_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>tau</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonSetRange">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>858</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Range</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxRange">
   <property name="geometry">
    <rect>
     <x>321</x>
     <y>858</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxInputParam">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>561</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Model Input Parameters</string>
   </property>
   <widget class="QLabel" name="labelInputParam1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R (Motion blur correction)</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxInputParam1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="maximum">
     <double>0.250000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>0.166700000000000</double>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ModelFitWidget</class>
   <extends>QGraphicsView</extends>
   <header>gui.tab.msd</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
